Lesson7/organizer7/scheduler/models.py

```diff
@@ -0,0 +1,22 @@
+# scheduler/models.py
+from django.db import models
+
+
+class User(models.Model):
+    objects = None
+    name = models.CharField(max_length=100)
+    password = models.CharField(max_length=100)
+    language = models.CharField(max_length=100)
+    grade = models.CharField(max_length=10)
+
+    def __str__(self):
+        return self.name
+
+
+class Note(models.Model):
+    objects = None
```

Comment:
This is not needed as well.

Reply:
So it's an issue with PyCharm Commutity Edition not having normal support of Django.

https://stackoverflow.com/questions/43865989/unresolved-attribute-reference-objects-for-class-in-pycharm